src/shared/components/empty-list/empty-list.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component, HostBinding, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-empty-list',
+  templateUrl: './empty-list.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EmptyListComponent {
+  @Input()
+  emptyListName: string;
+
+  @HostBinding('class')
+  private readonly classes = 'mx-auto';
+
+  buildTranslationKey(link: string, relativeKey: string): string {
```

Comment:
you don't need to use the `link: string` param here and pass it from the template, when you have access to it inside the class by calling `this.emptyListName` instead of it.